{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -161,3 +161,16 @@ build_command = "python -m pip install build; python -m build"
 
 [tool.setuptools]
 include-package-data = true
+
+
+[tool.tox]
+legacy_tox_ini = """
+[tox]
+envlist = py{39,310}
```

Comment:
Why don't we use this instead of the tags for denoting python versions (I would expect to change things here not in the tags)

Reply:
Just to be certain I understand; when you say "tags", I assume you mean e.g. `"Programming Language :: Python :: 3.9"`.

Ideally, I would like only **one** source of truth for supported versions. The closest we can get right now is the tags + the matrix in github actions. 

But I can see how this is currently counterintuitive. I'm not sure how to get those aligned.

Which solution do you think is preferable?

Reply:
Hmm hard one. 

I would say we need one place to decide on what to test on. Is that the envlist? and one place to denote what the package can run on. Ideally those should be 1-1. Which they mostly are now? (we mostly disregard the envlist?)

Reply:
Yeah, we actually need to signal many things:
1. Which envs the package can run on (currently in `pyproject.toml`)
2. Which envs `inv test` runs (currently parses `pyproject.toml`)
3. Which envs `tox` runs by default (currently in the `envlist`)
4. Which envs are run on github actions (currently in `tests.yml`)

Since we basically don't want people running `tox`, I think we can ignore that. Then it's 1:1 with the exception of `test.yml`. There's a trade-off here. 

If we have it run `inv test`, we could parse all the correct environments from the toml. That's nice 👍 
But then we can't use the github actions matrix, so each environment would run sequentially, dramatically increasing test run durations. Not as nice :-(

I think the current situation is pretty good, and really want tox to start moving out to the projects, so merging now. If you come up with a good solution to the above, I'm up for creating the PR 👍

Reply:
That is understandable (that is also why the PR was approved)